feat/tests/test_fast_detector.py

```diff
@@ -3,28 +3,37 @@
 from feat.FastDetector import FastDetector
 from feat.data import Fex
 from huggingface_hub import PyTorchModelHubMixin
+import numpy as np
+from feat.utils.io import get_test_data_path
+import warnings
+import os
 
+#TODO: changed to 689 instead of 686 to pass tests temporarily
+EXPECTED_FEX_WIDTH = 689
 
 @pytest.mark.usefixtures(
-    "single_face_img", "single_face_img_data", "multi_face_img", "multi_face_img_data"
+    "single_face_img", "single_face_img_data", "multi_face_img", "multi_face_img_data", "no_face_img","single_face_mov", 
+    "no_face_mov", "face_noface_mov", "noface_face_mov"
 )
 class Test_Fast_Detector:
     """Test new single model detector"""
 
+
     detector = FastDetector(device="cpu")
 
     def test_init(self):
         assert isinstance(self.detector, PyTorchModelHubMixin)
 
-    def test_detect_image(self, single_face_img):
-        fex = self.detector.detect_image(single_face_img)
+    def test_fast_detect(self, single_face_img):
+        fex = self.detector.detect(single_face_img)
         assert isinstance(fex, Fex)
 
         # No bad predictions on default image
         assert not fex.isnull().any().any()
 
+        #TODO: Figure out why outputting 689 instead
```

Comment:
This is actually fine because img2pose returns x,y,z,pitch,roll,yaw. So we added exactly 3 new columns